cmd/veeam-sos-api.go

```diff
@@ -143,17 +145,39 @@ func veeamSOSAPIGetObject(ctx context.Context, bucket, object string, rs *HTTPRa
 		if objAPI == nil {
 			return nil, errServerNotInitialized
 		}
-		info := objAPI.StorageInfo(ctx)
-		info.Backend = objAPI.BackendInfo()
 
-		usableTotal := int64(GetTotalUsableCapacity(info.Disks, info))
-		usableFree := int64(GetTotalUsableCapacityFree(info.Disks, info))
+		q, err := globalBucketQuotaSys.Get(ctx, bucket)
+		if err != nil {
+			return nil, err
```

Comment:
It seems globalBucketQuotaSys.Get() returns BucketQuotaConfigNotFound{} if there is not configuration, shouldn't we ignore this and continue ?

Reply:
Oh I see if it does let me change it @vadmeste